account_bank_statement_import_txt_xlsx/models/account_bank_statement_import_sheet_parser.py

```diff
@@ -56,6 +56,12 @@ def parse(self, mapping, data_file, filename):
             journal.code,
             path.basename(filename),
         )
+        if mapping.delete_rows > 0 and name.lower().endswith('.csv'):
```

Comment:
What's wrong with skipping XLS rows?

Reply:
agree, it is better to skip if possible. Was there any impediment to skip?